docs/ManualDeploymentGuideScalarDLAuditorOnAWS.md

```diff
@@ -52,35 +52,13 @@ so you need to add peering for internal communication between the Auditor, Ledge
 * You must create peering between Ledger, Auditor, and Client VPCs to enable communication between private networks.
 * You must update Ledger, Auditor, and Client route tables for VPC peering connections.
 
-### Recommendations
-
-* You should create Network ACLs for Ledger and Auditor VPCs.
-* You should set up the Network ACLs of the Ledger and Auditor to restrict unused accesses.
-
 ### Steps
 
 * Create 3 peering connections between the Ledger, Auditor and Client VPCs based on the [AWS official guide](https://docs.aws.amazon.com/vpc/latest/peering/create-vpc-peering-connection.html).
     * Peering between Ledger and Auditor.
     * Peering between Ledger and Client.
     * Peering between Auditor and Client.
 * Update route tables for VPC peering connection based on the [AWS official guide](https://docs.aws.amazon.com/vpc/latest/peering/vpc-peering-routing.html).
-* Add new Inbound and Outbound rule to restrict unwanted access to Ledger and Auditor.
-    * Add new Inbound rules to the Ledger Network ACLs to allow the Ledger Envoy LoadBalancer (e.g., 50051 and 50052 by default) access from the Auditor and Client.
-        * You must set high priority to this rule.
-    * Add new Inbound rules to the Ledger Network ACLs to allow ssh access from the internet.
-        * You must set second priority to this rule.
-    * Add new Inbound rules to Ledger Network ACLs to allow all access to Ledger VPC.
-        * You must set third priority to this rule.
-    * Add new Outbound rules to the Ledger Network ACLs to allow all access to the internet.
-    * Add Ledger private subnets to subnet associations of the Ledger Network ACLs.
-    * Add new Inbound rules to the Auditor Network ACLs to allow the Auditor Envoy LoadBalancer (e.g., 40051, 40052 by default) access from the Ledger and Client.
-        * You must set high priority to this rule.
-    * Add new Inbound rules to the Auditor Network ACLs to allow ssh access from the internet.
-        * You must set second priority to this rule.
-    * Add new Inbound rules to Auditor Network ACLs to allow all access to Auditor VPC.
-        * You must set third priority to this rule.
-    * Add new Outbound rules to the Auditor Network ACLs to allow all access to the internet.
-    * Add Auditor private subnets to subnet associations of the Auditor Network ACLs.
```

Comment:
@kota2and3kan san,
By default security group enables only Ledger and Auditor Envoy LoadBalancers while we using the Scalar DL manual deployment guide.
Security group for Ledger
<img width="1082" alt="security-group-ledger" src="https://user-images.githubusercontent.com/58680313/162342643-e8a81bef-2f84-4109-a072-842246871787.png">
Security group for Auditor
<img width="1091" alt="security-group-auditor" src="https://user-images.githubusercontent.com/58680313/162342628-7a4429bf-8443-4639-b255-b6c9ad6e5501.png">

Above opened ports will be resolved in the load balancer.

The above suggestions may be helpful to enable more restrictions or more customization, so I will create another PR for integrating the above suggestions.

In my understanding, the above suggestions are required for enabling advanced security features. By default, it will provide decent restrictions.

Could you please correct me, If my understanding is wrong?

Reply:
Your understading is correct. But, I have a minor question about default security groups.
It seems that the default security groups configuration is a bit different between your test environment and my test environmet.
I think it depends on the configuration of EKS and how to create EKS (what tools we use).

So, the difference is not problem, but I want to know how you create EKS cluster, for the reference.
Could you please tell me how you create EKS cluster (what tool you use)?
(* For your information, I use eksctl to create EKS cluster.)

Reply:
@kota2and3kan san,

> Could you please tell me how you create EKS cluster (what tool you use)?

I created the EKS cluster with the AWS management console.

I will create another PR for this.

Can I proceed?

Reply:
@scalar-boney How to create a cluster can be delegated to users (I think we should still provide recommendations) but what needs to be configured for security should be clearly specified in the doc.
Does the current doc meet the above?

Reply:
No, the security configuration-related details do not exist in the current document.
I will create a new PR for integrating the security configurations.

Reply:
OK, Yokoyama-san will update the English version.